StPetersburg/IVEf30.xml

```diff
@@ -40,9 +40,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         <msItem xml:id="ms_i1">
+                          <locus from="1r" to="30v"/>
                             <title type="complete" ref="LIT2352ShortC" cert="high"/>
                             <textLang mainLang="gez"/>
-                            <note>The text extends till <date when="1889"/> (till the accession of <persName ref="PRS7006Menilek"/>). The text resembles that of <ref type="mss" corresp="BNFet142"/> but is shorter.</note>
+                            <note>The narration continues till
+                              <date when="1889"/> (the year of <persName ref="PRS7006Menilek">Mǝnilǝk's</persName> accession
+                              to the throne as king of Ethiopia).
+                              The text resembles that of <ref type="mss" corresp="BNFet142"/> but is shorter,
+                              of inferior quality.</note>
```

Comment:
What does "of inferior quality" mean?

Reply:
It means that the text is of inferior quality than the BNFet142 (probably more mistakes, less careful hand etc. ). This was in the catalogue.